diff --git a/extra/mariabackup/xtrabackup.cc b/extra/mariabackup/xtrabackup.cc
index 7ae81fa97c6..d99fd0d79ec 100644
--- a/extra/mariabackup/xtrabackup.cc
+++ b/extra/mariabackup/xtrabackup.cc
@@ -6814,6 +6814,10 @@ static int get_exepath(char *buf, size_t size, const char *argv0);
 /* ================= main =================== */
 int main(int argc, char **argv)
 {
+    /* In case the program hangs testing tools (RQG etc.) using it
+    might need its process id for killing it. */
+    msg("Starting Mariabackup as process %lu", (ulong) getpid());
+
   char **server_defaults;
   char **client_defaults;
   char **backup_defaults;
