diff --git a/mysql-test/lib/My/SafeProcess/safe_process.cc b/mysql-test/lib/My/SafeProcess/safe_process.cc
index dcf9491d2d6..6a25d111962 100644
--- a/mysql-test/lib/My/SafeProcess/safe_process.cc
+++ b/mysql-test/lib/My/SafeProcess/safe_process.cc
@@ -149,7 +149,7 @@ static int kill_child(bool was_killed)
     kill(-child_pid, SIGTERM);
     sleep(10); // will be interrupted by SIGCHLD
     if (!(ret_pid= waitpid(child_pid, &status, WNOHANG)))
-      kill(-child_pid, SIGKILL);
+      kill(-child_pid, SIGABRT);
   }
 
   if (!ret_pid)
