diff --git a/storage/innobase/include/os0file.h b/storage/innobase/include/os0file.h
index e0f398301e3..9c86a182655 100644
--- a/storage/innobase/include/os0file.h
+++ b/storage/innobase/include/os0file.h
@@ -273,7 +273,7 @@ struct os_file_size_t {
 	os_offset_t	m_alloc_size;
 };
 
-constexpr ulint OS_AIO_N_PENDING_IOS_PER_THREAD= 256;
+constexpr ulint OS_AIO_N_PENDING_IOS_PER_THREAD= 1;
 
 extern Atomic_counter<ulint> os_n_file_reads;
 extern Atomic_counter<size_t> os_n_file_writes;
