diff --git a/sql/sql_backup.cc b/sql/sql_backup.cc
index 56d76aa10d0..b12e110f891 100644
--- a/sql/sql_backup.cc
+++ b/sql/sql_backup.cc
@@ -49,7 +49,7 @@ copy_step(int in_fd, int out_fd, size_t count, off_t *offset) noexcept
 {
   return copy_file_range(in_fd, offset, out_fd, offset, count, 0);
 }
-# define cfr(src,dst,start,end) copy<copy_step>(src, dst, start, end)
+//# define cfr(src,dst,start,end) copy<copy_step>(src, dst, start, end)
 #endif
 #ifdef __linux__
 # include <sys/sendfile.h>
